Runtime/HighLevel/StatementPromise.cs

```diff
@@ -20,6 +21,30 @@ public StatementPromise(Statement statement, Task<Statement> task)
             this.Promise = task;
         }
 
+        public StatementPromise addContextActivityParent(string id, string name, string description, string type) {
```

Comment:
This does not follow the project naming conventions should be always PascalCase

Reply:
Additionally it seems like youre not referencing this code in your PR. I suggest removing

Reply:
fixed for PascalCase but keep because can be needed outside Xasu by developers to add those Grouping context activities